### ISSUE TITLE
How to install to vscode

### ISSUE THREAD

**Issue by kythuatwebso** [TomaisLeannan/vscode-sftp-revived#6]
I can not find this extension at market vscode

![image](https://user-images.githubusercontent.com/57938377/106842244-3d280700-66d6-11eb-88ae-2c83e202d3aa.png)

**Comment by AndisGrossteins** [comment]
As @TomaisLeannan states in his announcement issue: liximomo/vscode-sftp#875
> I haven't released anything to the marketplace yet.

It's not yet published on the marketplace because it needs a lot of work.
However, you could build the extension yourself by fetching the v1.12.12 tag, which _*should*_ be stable.
I'm about to do it myself for the first time. If you or anyone else needs some pointers there's the documentation: https://code.visualstudio.com/api/working-with-extensions/publishing-extension
You can ignore the publishing part - what you want is the packaging steps.

After successfully packaging the extension, you should be able to load the *.vsix file from extension manager pane menu `Install from VSIX...`.
![image](https://user-images.githubusercontent.com/212789/106852464-c7657080-66c0-11eb-9e1f-61f2c69fa139.png)

**Comment by TomaisLeannan** [comment]
@AndisGrossteins that is correct.  Also, most of the "improvements" made in 1.12.12 were cosmetic and syntactic.  The biggest update there were some dependencies being updated I believe.

I'm in the middle moving to a new house and getting a new job right now, which should conclude in about a week.  I'll be off and on sparingly until then.

**Comment by TomaisLeannan** [comment]
I'll keep this issue open until I get a version published to the Marketplace.

**Comment by mean-cj** [comment]
+1 for published  , thank you for your update sftp.

**Comment by thompsbrian** [comment]
No problem... getting back to it this week.

**Comment by thompsbrian** [comment]
Thank you, plus there's a lot, lot more lol.  Anyway, I have been using "Remote - SSH" by Microsoft in place of the original vscode-sftp extension by liximomo.  Doesn't look like I'll actually have any time to fix this plugin or maintain it.

Check out the plugin I mentioned and see if it meets your needs.

**Comment by jeremy-farrance** [comment]
Just making this a bit more visible to casual visitors, skimmers-in-a-hurry.

> # Doesn't look like I'll actually have any time to fix this plugin or maintain it.
> _- thompsbrian_
